plugins/generic/usageStats/locale/en_US/locale.xml

```diff
@@ -34,6 +34,8 @@
 	<message key="plugins.generic.usageStats.settings.compressArchives.description">Activate this option to compress archived log files using gzip tool (you will have to configure the gzip setting inside config.inc.php). If you have a high traffic site it's a good idea to enable this, so you can save some extra disk space.</message>
 	<message key="plugins.generic.usageStats.settings.compressArchives">Compress archives</message>
 	<message key="plugins.generic.usageStats.settings.statsDisplayOptions">Statistics display options</message>
+	<message key="plugins.generic.usageStats.settings.statsDisplayOptions.contextWide">The following display settings apply only to the current context:</message>
+	<message key="plugins.generic.usageStats.settings.statsDisplayOptions.siteWide">The following display settings will be used as default i.e. for each context that does not overwrite them:</message>
```

Comment:
Do you maybe have a suggestion, how to make this text OJS and OMP neutral i.e. without 'journal' or 'press'? Can we use context?